.github/AGENTS.md

```
@@ -0,0 +1,312 @@
+ctest --test-dir cpp/build
+
+# Run Python tests
+pytest -v python/cuopt/cuopt/tests
+
+# Debug build
+./build.sh libcuopt -g
+```
+
+---
+
+*Last updated: December 2024 | cuOpt v25.12*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)